services/baseremotecontrol/builtin/builtin.go

```diff
@@ -574,26 +581,3 @@ func (ts *throttleState) init() {
 		input.AbsoluteHat0Y: 0.0,
 	}
 }
-
-func parseEvent(mode controlMode, state *throttleState, event input.Event) (r3.Vector, r3.Vector) {
-	state.mu.Lock()
-	defer state.mu.Unlock()
-
-	newLinear := state.linearThrottle
-	newAngular := state.angularThrottle
-
-	switch mode {
-	case joyStickControl:
-		newLinear.Y, newAngular.Z = oneJoyStickEvent(event, state.linearThrottle.Y, state.angularThrottle.Z)
-	case droneControl:
-		newLinear, newAngular = droneEvent(event, state.linearThrottle, state.angularThrottle)
-	case triggerSpeedControl:
-		newLinear.Y, newAngular.Z = triggerSpeedEvent(event, state.linearThrottle.Y, state.angularThrottle.Z)
-	case buttonControl:
-		newLinear.Y, newAngular.Z, state.buttons = buttonControlEvent(event, state.buttons)
-	case arrowControl:
-		newLinear.Y, newAngular.Z, state.arrows = arrowEvent(event, state.arrows)
-	}
-
-	return newLinear, newAngular
-}
```

Comment:
This is ONLY used in a test... and the test is only testing this function itself. Pointless.